modules/microshift-auto-apply-manifests.adoc

```diff
@@ -76,43 +75,43 @@ spec:
         command:
           - sleep
           - "3600"
+EOF
 ----
-// what are the "---" after name: busybox?
 .Procedure
 . Create the `kustomize` manifest files by running the following commands:
 +
-* Run the command to read and write the standard inputs.
-[source,terminal]
+.. Place the YAML file in the directory:
 +
+[source,terminal]
 ----
-$ sudo tee
+$ sudo cat << EOF > ${MANIFEST_DIR}/kustomization.yaml
```

Comment:
We'll want to come back to fix up these commands. This cat and the YAML content below need to all be together in 1 box so the user can copy the whole command and its input. I'll help do that in another PR, after this lands.